source/playbooks/CIS140/ssmdocs/CIS140_2.1.5.1.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { Construct } from 'constructs';
+import { PlaybookProps } from '../../SC/lib/control_runbooks-construct';
+import { ControlRunbookDocument } from '../../SC/ssmdocs/control_runbook';
+import { ConfigureS3PublicAccessBlockDocument } from '../../SC/ssmdocs/SC_S3.1';
+
+export function createControlRunbook(stage: Construct, id: string, props: PlaybookProps): ControlRunbookDocument {
+  return new ConfigureS3PublicAccessBlockDocument(stage, id, { ...props, controlId: '2.1.5.1' });
```

Comment:
by my reading, 2.1.5.1 is referring to the account-level and 2.1.5.2 is referring to the bucket-level public access settings, so is it correct for them to both use the same remediation? Does this runbook handle the difference between bucket and account level remediations transparently?

Reply:
They don't. 2.1.5.1 uses `ConfigureS3PublicAccessBlockDocument` and 2.1.5.2 uses `ConfigureS3BucketPublicAccessBlockDocument`, which looks correct to me.

Reply:
I reread them 3 times to confirm they were actually the same, I promise...